posix.mak

```diff
@@ -170,7 +170,7 @@ PACKAGE_std = array ascii base64 bigint bitmanip compiler complex concurrency \
   outbuffer parallelism path process random signals socket socketstream stdint \
   stdio stdiobase stream string system traits typecons typetuple uni \
   uri utf uuid variant xml zip zlib
-PACKAGE_std_experimental = typecons
+PACKAGE_std_experimental = checkedint typecons
```

Comment:
it's annoying and we should really get rid off these two additional files, but until we do you need to update `win32.mak` & `win64.mak` too

Btw [see this thread](http://forum.dlang.org/post/wckshfgkltmmcgmbbitu@forum.dlang.org) for the current status.